keylime.conf

```diff
@@ -127,3 +127,7 @@ tpm_signing_alg = rsassa
 # handle (e.g. "0x81000000"). The Keylime agent will then not attempt to
 # create a new EK upon startup, and neither will it flush the EK upon exit.
 ek_handle = generate
+
+# Use this option to state the existing TPM ownerpassword. This option should
+# be set only when ek_handle option points to an existing EK.
+tpm_ownerpassword =
```

Comment:
+1 , might even be worth removing entirely.

Reply:
> As this option is optional, maybe we could comment it out rather than having an empty value?

Could you please clarify under which circumstances the password is optional? What is the expected TPM setup?
Let's say I start with TPM after running `tpm2_clear`. What an administrator is supposed to do so that the rust agent would work properly?

Reply:
Please see https://github.com/keylime/rust-keylime/issues/429. My understanding is the password would only be used if the Rust agent needs to use `tpm2_changeauth` to reset after another program takes TPM ownership. The Rust agent should generally work properly without taking any ownership or setting any passwords for the TPM.

Reply:
So the password is not necessary to generate EK? I will try doing that manually tomorrow.

Reply:
I could only generate EK on a TPM which had the password set (ownership taken) by using an authenticated session.
This is the case, for example, on `keylime/keylime` test suite as during the test it takes ownership and sets the password as `keylime`.

Reply:
Would this be a case where the test should be updated, since this is functionality that was needed for the Python agent only? I could be missing something, but we were transitioning away from this. I can try to find out more as well.

Reply:
@THS-on commented above "Note that the generate option should be not implemented in the rust agent, because there is no reason to do that" - so do we need to do this?

Reply:
@lkatalin The rust agent always generated the EK as far as I remember. It is true that if correctly provisioned, the TPM should not generate the EK and the handle should be set via the `ek_handle` configuration option. But note that the support for doing this was recently introduced by @ueno via #426

Reply:
@lkatalin About the other question, it is not about changing the test, but the python agent implementation itself. In my opinion, it should stop taking ownership of the TPM.

Reply:
@ansasaki I see, since we are not yet removing the Python agent completely, it is true that the Rust agent will in most cases need to conform to the existing tests. Re: generating the EK, I don't have an opinion on this but just want to make sure everyone agrees. Perhaps it can be a separate issue.